freud/environment.pyx

```diff
@@ -566,25 +575,27 @@ cdef class EnvironmentCluster(_MatchEnv):
                 neighbor pairs to use in the calculation, or a dictionary of
                 `query arguments
                 <https://freud.readthedocs.io/en/stable/topics/querying.html>`_
-                (Default value: None).
+                (Default value: None). Defines the particle neighborhoods for
+                comparing environments.
             env_neighbors (:class:`freud.locality.NeighborList` or dict, optional):
                 Either a :class:`NeighborList <freud.locality.NeighborList>` of
                 neighbor pairs to use in the calculation, or a dictionary of
-                `query arguments
+                `query arguments.
                 <https://freud.readthedocs.io/en/stable/topics/querying.html>`_
```

Comment:
I'm not asking to remove the period, I'm asking to move it outside the hyperlink. I think this will keep the period in the rendered documentation, but keep it from becoming highlighted purple once the link is clicked.